tools/calyx-pass/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "calyx-pass"
```

Comment:
How about `calyx-pass-explorer` instead?

Reply:
Sigh. I guess I have to accept the un-TAB-ability :(